src/shapes/Object/defaultValues.ts

```diff
@@ -36,75 +36,48 @@ export const cacheProperties = [
 ];
 
 export const fabricObjectDefaultValues = {
-  originX: LEFT,
-  originY: TOP,
+  // coming from ObjectOrigin
   top: 0,
   left: 0,
   width: 0,
   height: 0,
-  scaleX: 1,
-  scaleY: 1,
   flipX: false,
   flipY: false,
-  opacity: 1,
-  angle: 0,
+  scaleX: 1,
+  scaleY: 1,
   skewX: 0,
   skewY: 0,
-  cornerSize: 13,
-  touchCornerSize: 24,
-  transparentCorners: true,
-  hoverCursor: null,
-  moveCursor: null,
+  originX: LEFT,
+  originY: TOP,
+  angle: 0,
+  strokeWidth: 1,
+  strokeUniform: false,
+  // coming from ObjectGeometry
   padding: 0,
-  borderColor: 'rgb(178,204,255)',
-  borderDashArray: null,
-  cornerColor: 'rgb(178,204,255)',
-  cornerStrokeColor: '',
-  cornerStyle: 'rect',
-  cornerDashArray: null,
-  centeredScaling: false,
-  centeredRotation: true,
+  // coming from AnimatableObject
+  colorProperties: ['fill', 'stroke', 'backgroundColor'],
```

Comment:
this should be a static prop or even not a prop

Reply:
Made static